open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/pom.xml

```diff
@@ -37,6 +37,11 @@
 	<dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
+    </dependency>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
+            <artifactId>ui-admin-services-api</artifactId>
+            <version>1.1-SNAPSHOT</version>
```

Comment:
A version should not be specified here - we only put versions in the top level pom.
Need new issue

Reply:
If not in top level pom, add any new modules to the <dependencyManagement> section there
Ideally <scope> I think should be 'runtime' for all our spring components not references in the code, but discovered during runtime component scanning